src/searches/useCategorySearch.ts

```diff
@@ -14,6 +14,14 @@ export const searchCategories = gql`
         node {
           id
           name
+          ancestors(first: $first) {
```

Comment:
you pass here the same variable as for the root query (categories), which is wrong, this number should be either configured separately or even hardcoded so now the number of loaded categories indicates also amount of ancestors, which is not really expected here: having eg first 2 categories I still want eg. to have 3 ancestors (not 2)

**Consider:**
- keep number of ancestors as separate variable
- search categories query is used in several places, do you need ancestors in each? perhaps a separated query is more reasonable here? (think about it if you have time)